cypress/integration/home.spec.ts

```diff
@@ -26,6 +26,24 @@ it('should display corresponding information', () => {
 
   cy.contains('footer', 'Created with')
   cy.contains('footer', 'by Mario')
+
+  cy.findByRole('link', { name: /mario/i }).should(
+    'have.attr',
+    'href',
+    'https://mario.dev/'
+  )
+
+  cy.findByRole('link', {
+    name: /octoclairvoyant repository on github/i,
+  }).should(
+    'have.attr',
+    'href',
+    'https://github.com/octoclairvoyant/octoclairvoyant-webapp'
+  )
+
+  cy.findByRole('img', { name: /powered by vercel logo/i })
+
+  // TODO: Add test that checks if the Vercel img has the correct href, should be: https://vercel.com/?utm_source=octoclairvoyant-team&utm_campaign=oss
```

Comment:
The thing that's missing here is the assertion that checks if the Vercel `<img>` has the correct `href`. I still don't know how to check for this, so that's whats holding up this PR from getting merged.

Reply:
Aah, I see. The image itself doesn't have any `href` attribute, so we need to query the link for that. Since the image is inside the link, the best strategy is to query the link and assert whatever we want, and then query the image inside the link element obtained before and assert whatever we want for the image. If we remove the previous assert to find the Vercel logo, it can be replaced with this:

```js
// we first query the link
cy.findByRole('link', { name: /powered by vercel/i })
    // and check it has the proper href
    .should(
      'have.attr',
      'href',
      'https://vercel.com/?utm_source=octoclairvoyant-team&utm_campaign=oss'
    )
    // now, since `should` returns a Chainable, we can chain it with `within`, which allows us to query
    // for something inside the previous element
    .within(() => {
      // the image is gonna be searched within the link, so we can make sure the logo is part of the link!
      cy.findByRole('img', { name: /powered by vercel logo/i })
    })
```

Reply:
Ah that works! I didn't know about the `.within()` thing! 😄 

Thanks for the comments, I've left them in the production code, as I will probably need to refer to this later on for other tests... 🙈 😄